operator/controllers/common/phasehandler.go

```diff
@@ -41,7 +41,7 @@ func (r PhaseHandler) HandlePhase(ctx context.Context, ctxAppTrace context.Conte
 	piWrapper.SetCurrentPhase(phase.ShortName)
 
 	r.Log.Info(phase.LongName + " not finished")
-	ctxAppTrace, spanAppTrace, err := r.SpanHandler.GetSpan(ctxAppTrace, tracer, reconcileObject, phase.ShortName)
+	_, spanAppTrace, err := r.SpanHandler.GetSpan(ctxAppTrace, tracer, reconcileObject, phase.ShortName)
```

Comment:
pls rename, this is not spanAppTrace but a general trace in PhaseHandler

Reply:
at the moment we have two traces that are running in parallel. One describes the application deployment `spanAppTrace` and one describing the lifecycle toolkit internal execution. 
I would argue to keep the name as is since the trace we are touching describes the App Deployment lifecycle